licenses/licenses.json

```diff
@@ -1,10 +1,2751 @@
 {
-    "EasyBuild": {
-        "spdx": "GPL-2.0-only",
-        "license_url": "https://easybuild.io"
+    "ALL/0.9.2-foss-2023a": {
+        "License": "not found",
+        "Needs Manual Check": true,
+        "Retrieved From": "not found"
     },
-    "GCCcore": {
-        "spdx": "GPL-2.0-with-GCC-exception",
-        "license_url": "https://github.com/gcc-mirror/gcc/blob/master/COPYING"
+    "AOFlagger/3.4.0-foss-2023b": {
+        "License": "not found",
+        "Needs Manual Check": true,
+        "Retrieved From": "not found"
+    },
+    "ASE/3.22.1-gfbf-2022b": {
+        "License": "not found",
+        "Needs Manual Check": true,
+        "Retrieved From": "not found"
+    },
+    "ATK/2.38.0-GCCcore-13.2.0": {
+        "License": [
+            "Other"
```

Comment:
How did this end up with `Other`? I see it is GPL v2 at https://gitlab.gnome.org/Archive/atk